format/FlightSql.proto

```diff
@@ -1416,14 +1477,47 @@ message ActionCreatePreparedStatementRequest {
 
```

Comment:
Do you plan to remove the "experimental" markers like the above?

Reply:
I think not in this RFC, but we should do it in the near future (perhaps after the JDBC driver has seen some use)